src/hedera_sdk_python/timestamp.py

```diff
@@ -72,7 +74,7 @@ def to_date(self) -> datetime:
         Returns:
             datetime: A `datetime` instance.
         """
-        return datetime.fromtimestamp(self.seconds) + timedelta(
+        return datetime.fromtimestamp(self.seconds, tz=timezone.utc) + timedelta(
```

Comment:
This change was needed to fix Windows-specific issues with timestamp parsing.

As far as I see, it should not cause any issues + tests are running fine on MacOS as well.